students/kristianjf/lesson09/mailroom_5.py

```diff
@@ -12,44 +12,68 @@
 the amounts they have donated. This structure should be populated at first with at \
 least five donors, with between 1 and 3 donations each.
 '''
-# self.DATA_STRUCTURE = [('Jeff Bezos', [200, 500, 1000]), \
-#                     ('Bill Gates', [1000]), \
-#                     ('Steve Jobs', [20, 50, 100])]
-# self.DATA_STRUCTURE_DICTS = [{'name': name, 'donation': donation} for name, donation in self.DATA_STRUCTURE]
-# # self.DATA_STRUCTURE_DICT = [ \
-# #                         {'name': 'Jeff Bezos', 'donation': [200, 500, 1000]},\
-# #                         {'name': 'Bill Gates', 'donation': [1000]},\
-# #                         {'name': 'Steve Jobs', 'donation': [20, 50, 100]}\
-# #                         ]
-
 class Donor:
     def __init__(self, name, donations):
         self._name = name
-        self._donations = [] if donations is None else donations
+        self._normalized_name = ''
+        self._donations = donations if isinstance(donations, list) else [donations]
+    def __lt__(self, other):
+        return sum(self.donations) < sum(other.donations)
     @property
     def name(self):
         return self._name
     @property
     def donations(self):
         return self._donations
+    @property
+    def normalized_name(self):
+        'Validate and Return Name Input'
```

Comment:
Use proper docstring commenting delimiters (`'''...'''`)

Reply:
good note -- technically, a docstring is just a string, so single quotes work, but it's considered good style to use triple quotes even if it's a one-liner docstring -- that way it's easier to extend.